index.html

```diff
@@ -206,5 +206,78 @@ <h4>Skills</h4>
         
     </section>
 </section>
+
+<footer>
+        <div class="footspace"></div> 
+        <div class=" box namet"><h2>Get Started</h2></div>
+        <div class=" box introt"><p>Lorem ipsum dolor sit amet,
+             consectetur adipiscing elit, sed do eiusmod tempor
+              incididunt ut labore et dolore magna.</p></div>
```

Comment:
- [x] Great job so far! But unfortunately as pointed by the previous reviewer, you need to stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to give the title and paragraph the right size so they can look closely similar to the [original design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471) 😸 .

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/149048688-539c048d-74f2-4913-84a7-14ec96a4840d.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/149048708-372d1bd4-1c5d-4425-af7e-a50f27b75d59.PNG)